RAW_RELEASE_NOTES.md

```diff
@@ -0,0 +1,10 @@
+# Raw release notes
+
+This file contains "raw" release notes for each release. The notes are added by developers as changes
+are made to Envoy that are user impacting. When a release is cut, the releaser will use these notes
+to populate the [Sphinx release notes in data-plane-api](https://github.com/envoyproxy/data-plane-api/blob/master/docs/root/intro/version_history.rst).
+The idea is that this is a low friction way to add release notes along with code changes. Doing this
+will make it substantially easier for the releaser to "linkify" all of the release notes in the
+final version.
```

Comment:
I think we can populate this from the git commit logs if we are diligent about ensuring that folks include the release notes field (and make sure it's machine parseable).